openedx/core/djangoapps/notifications/base_notification.py

```diff
@@ -282,9 +284,13 @@ def get_notification_content(notification_type, context):
     """
     Returns notification content for the given notification type with provided context.
     """
+    html_tags_context = {
+        tag: tag
+        for tag in ['strong', 'p']
```

Comment:
Mapping is constant so it would be nice if it is a pre-defined dict not generated one. 

i.e 
```python
html_tags_context =  {
    'p':'p'
    'strong': 'strong',
}
```

Reply:
Agree with Ahtisham